pkg/driver/clients/vhivemetadata.go

```diff
@@ -0,0 +1,58 @@
+package clients
+
+import (
+	"encoding/json"
+	"time"
+
+	ctrdlog "github.com/containerd/log"
+	"github.com/sirupsen/logrus"
+)
+
+type vHiveMetadata struct {
+	WorkflowId   string    `json:"WorkflowId"`
+	InvocationId string    `json:"InvocationId"`
+	InvokedOn    time.Time `json:"InvokedOn"`
+}
```

Comment:
I guess there is a copy of this structure somewhere in the vSwarm repo. Can you import it directly from there so we avoid duplicating structures and maintain consistency?

Reply:
Yes, this is the same file, hut importing it from vSwarm directly had some issues with the go.mod file since this isn't directly declared as a package under the vSwarm repo

Reply:
I would give this another try?